src/backend/dx12/src/command.rs

```diff
@@ -198,19 +311,119 @@ impl CommandBuffer {
             );
         }
     }
+
+    fn set_gr_bind_point(&mut self) {
+        if self.active_bindpoint != BindPoint::Graphics {
+            // Switch to graphics bind point
+            assert!(self.gr_pipeline.pipeline.is_some(), "No graphics pipeline bound");
+            let (pipeline, _) = self.gr_pipeline.pipeline.unwrap();
```

Comment:
just doing `expect` would be more idiomatic (than checking for `is_some()` in an `assert!`)